kotlin-mui-base/src/jsMain/generated/mui/base/Button.classes.kt

```diff
@@ -17,3 +19,5 @@ external interface ButtonClasses {
     /** State class applied to the root `button` element if `focusVisible={true}`. */
     var focusVisible: ClassName
 }
+
+external val buttonClasses: ButtonClasses
```

Comment:
In new scheme `ButtonClasses` properties must be readonly (`val`s), musn't be?
Also `ButtonClasses` must be `sealed` - inheritance doesn't have big sense.

Reply:
Looks like separate change, isn't it?

Reply:
If you update `ButtonClasses` usage, please fix contract too.
Otherwise  `buttonClasses` declaration will be invalid (if it's readonly).

Reply:
Done